nix/supervisord-cluster/tests/base.py

```diff
@@ -1,13 +1,14 @@
-"""Usage: cluster-test.py [--network-magic=MAGIC] [--state-dir=DIR]
+"""Usage: cluster-test.py [--network-magic=INT] [--state-dir=DIR] [--num-delegs=INT]
 
 Options:
     -n, --network-magic <magic>  network magic [default: 42]
```

Comment:
<magic> --> <int> to match L1 and L5?

Reply:
yeah, the change doesn't really do anything, just changed it to the type of it instead of some other description.